rules/hmap.bzl

```diff
@@ -113,7 +86,7 @@ headermap = rule(
             mandatory = False,
             doc = "Targets whose direct headers should be added to the list of hdrs",
         ),
-        "flatten_headers": attr.bool(
+        "namespace_headers": attr.bool(
```

Comment:
I wonder if we could remove this, and just namespace when a `namespace` is passed?

Reply:
Yeah, that's a good idea. I'll pick it up after we merge #61